schema.json

```diff
@@ -0,0 +1,68 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "message": {
+      "type": "string",
+      "description": "Message of the changelog",
+      "minLength": 1,
+      "maxLength": 1000
+    },
+    "type": {
+      "type": "string",
+      "description": "Changelog type",
+      "enum": [
+        "feature",
+        "bugfix",
+        "dependency",
+        "deprecation",
+        "breaking_change",
+        "performance"
+      ]
+    },
+    "scope": {
+      "type": "string",
+      "description": "Changelog scope",
+      "enum": [
+        "Core",
+        "Plugin",
+        "PDK",
+        "Admin API",
+        "Performance",
+        "Configuration",
+        "Clustering",
+        "Portal"
+      ]
+    },
+    "prs": {
+      "type": "array",
+      "description": "List of associated GitHub PRs",
+      "items": {
+        "pattern": "^(\\d+|\\w+\/\\w+:\\d+)$",
+        "type": ["integer", "string"],
+        "examples": ["1", "torvalds/linux:1"]
+      }
+    },
+    "issues": {
+      "type": "array",
+      "description": "List of associated GitHub issues",
+      "items": {
+        "pattern": "^(\\d+|\\w+\/\\w+:\\d+)$",
+        "type": ["integer", "string"],
+        "examples": ["1", "torvalds/linux:1"]
```

Comment:
I don't think we will ever support linking PRs outside of the org. AFAIK no one has used the `torvalds/linux:1` format up until now. Is this really needed?

Reply:
Done in https://github.com/Kong/gateway-changelog/pull/2